Solutions/Menes.PetStore.Specs/Steps/Steps.cs

```diff
@@ -0,0 +1,209 @@
+            Assert.AreEqual(expectedValue, actualValue);
+        }
+
+        [Then("the response object should have a property called '(.*)'")]
+        public void ThenTheResponseObjectShouldHaveAPropertyCalled(string propertyPath)
+        {
+            this.GetExpectedTokenFromResponseObject(propertyPath);
+        }
+
+        [Then("the response object should have a string property called '(.*)' with value '(.*)'")]
+        public void ThenTheResponseObjectShouldHaveAStringPropertyCalledWithValue(string propertyPath, string expectedValue)
+        {
+            JToken actualToken = this.GetExpectedTokenFromResponseObject(propertyPath);
+
+            string actualValue = actualToken.Value<string>();
+            Assert.AreEqual(expectedValue, actualValue);
+        }
+
+        [Then("the response object should not have a property called '(.*)'")]
+        public void ThenTheResponseObjectShouldNotHaveAPropertyCalled(string propertyPath)
+        {
+            JObject data = this.scenarioContext.Get<JObject>();
+            JToken token = data.SelectToken(propertyPath);
+            Assert.IsNull(token);
+        }
+
+        [Then(@"the response object should have an array property called '(.*)' containing (.*) entries")]
+        public void ThenTheResponseObjectShouldHaveAnArrayPropertyCalledContainingEntries(string propertyPath, int expectedEntryCount)
+        {
+            JToken actualToken = this.GetExpectedTokenFromResponseObject(propertyPath);
+            JToken[] tokenArray = actualToken.ToArray();
+            Assert.AreEqual(expectedEntryCount, tokenArray.Length);
+        }
+
+        private async Task SendGetRequest(string path)
+        {
+            HttpResponseMessage response = await HttpClient.GetAsync(new Uri(BaseUri, path)).ConfigureAwait(false);
+
+            this.scenarioContext.Set(response);
+
+            string content = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+
+            if (!string.IsNullOrEmpty(content))
+            {
+                Console.WriteLine($"Read response content: '{content}'");
+
+                var data = JObject.Parse(content);
+
+                this.scenarioContext.Set(data);
+            }
+        }
+
+        private async Task SendPostRequest(string path, object body)
+        {
+            string requestContentString = JsonConvert.SerializeObject(body);
+            var requestContent = new StringContent(requestContentString, Encoding.UTF8, "application/json");
+
+            HttpResponseMessage response = await HttpClient.PostAsync(new Uri(BaseUri, path), requestContent).ConfigureAwait(false);
+
+            this.scenarioContext.Set(response);
+
+            string responseContent = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+
+            if (!string.IsNullOrEmpty(responseContent))
+            {
+                Console.WriteLine($"Read response content: '{responseContent}'");
```

Comment:
Just checking: is this `Console.WriteLine` meant to be in here?

Reply:
Not really, no.